subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/DefaultLenientConfiguration.java

```diff
@@ -238,7 +238,7 @@ public void run(BuildOperationContext context) {
                 String displayName = "Resolve files of " + configuration.getIdentityPath();
                 return BuildOperationDescriptor
                     .displayName(displayName)
-                    .progressDisplayName(displayName)
+                    .progressDisplayName("Resolving files of " + configuration.getIdentityPath())
```

Comment:
As above, this is probably going to break some tests:
https://github.com/gradle/gradle/blob/56fc50c36cadbda2ce69c2fdd1c9c4b023133422/subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/RemoteDependencyResolveConsoleIntegrationTest.groovy#L92-L92

And there are TAPI tests for this message too :/